src/routes/api/authRoutes.js

```diff
@@ -10,7 +10,7 @@ const { sendPasswordResetEmail, receiveNewPassword } = resetPassword;
 
 
 const route = Router();
-const { signup, fbgooglesignup, login, logout} = authControllers;
+const { signup, fbgooglesignup, login, logout, getUserProfile, updateProfile } = authControllers;
```

Comment:
why not use `import destructuring`

Reply:
I just followed the structure I met on ground.